app/javascript/react/store/thresholdSlice.ts

```diff
@@ -26,10 +26,10 @@ export interface ThresholdState {
 export const initialState: ThresholdState = {
   defaultValues: {
     min: 0,
-    low: 0,
-    middle: 0,
-    high: 0,
-    max: 0,
+    low: 9,
+    middle: 35,
+    high: 55,
+    max: 150,
```

Comment:
Why do we need to change defaults? I think this is creating the bug I mentioned earlier. If we want to keep it like that we need to improve conditions for loading thresholds
```
  useEffect(() => {
    if (!isFirstRenderForThresholds.current) {
      dispatch(resetUserThresholds());
    }
    // #DirtyButWorks :nervous-laugh: -> refactor when moving thresholds to url
    if (defaultThresholds.max !== 0) {
      isFirstRenderForThresholds.current = false;
    }
  }, [defaultThresholds]);

  useEffect(() => {
    if (isFirstRenderForThresholds.current) {
      dispatch(setUserThresholdValues(initialThresholds));
    }
    // #DirtyButWorks :nervous-laugh: -> refactor when moving thresholds to url
    if (initialThresholds.max === 0) {
      isFirstRenderForThresholds.current = false;
    }
  }, [initialThresholds]);